relay_handler.js

```diff
@@ -63,10 +63,14 @@ RelayHandler.prototype.handleLazily = function handleLazily(conn, reqFrame) {
     }
 
     if (self.circuits) {
-        self.logger.warn(
-            'circuit breaking for lazy relaying isn\'t implemented',
-            rereq.extendLogInfo({}));
-        return false;
+        var circuit = self.circuits.getCircuit(rereq.callerName, rereq.serviceName, rereq.endpoint);
```

Comment:
It's a lot easier to "verify" that this PR is "correct" if you fix eager to have the same code.